From: Michael Biebl <biebl@debian.org>
Date: Fri, 9 Dec 2011 00:31:21 +0100
Subject: [PATCH] Revert "Default to AdminIdentities=unix-group:wheel for
 local authority"

This reverts commit 763faf434b445c20ae9529100d3ef5290976d0c9.

On Red Hat derivatives, every member of group 'wheel' is necessarily
privileged. On Debian derivatives, there is no wheel group, and gid 0
(root) is not used in this way. Change the default rule to consider
uid 0 to be privileged, instead.

On Red Hat derivatives, 50-default.rules is not preserved by upgrades;
on dpkg-based systems, it is a proper conffile and may be edited
(at the sysadmin's own risk), so the comment about not editing it is
misleading.

[smcv: added longer explanation of why we make these changes]
---
 src/polkitbackend/50-default.rules | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/polkitbackend/50-default.rules b/src/polkitbackend/50-default.rules
index f427ae1..ef1215a 100644
--- a/src/polkitbackend/50-default.rules
+++ b/src/polkitbackend/50-default.rules
@@ -1,12 +1,10 @@
 /* -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*- */
 
-// DO NOT EDIT THIS FILE, it will be overwritten on update
-//
 // Default rules for polkit
 //
 // See the polkit(8) man page for more information
 // about configuring polkit.
 
 polkit.addAdminRule(function(action, subject) {
-    return ["unix-group:wheel"];
+    return ["unix-user:0"];
 });
